docs/_locale/zh_CN/LC_MESSAGES/stpl.po

```diff
@@ -101,7 +101,7 @@ msgstr "{{}}中的Python语句会在渲染的时候被执行，可访问传递
 
 #: ../../stpl.rst:62
 msgid "Embedded python code"
-msgstr "嵌入Pyhton代码"
+msgstr "嵌入Python代码"
```

Comment:
As with the German translation, this is a typo in the translation and I don't have the resources to send feedback otherwise.

Also note that while I've made these changes here, if I were to return at a later date, I wouldn't notice typos in the translations as I've added them to an exclusion list.